### DISCUSSION TITLE
jobrunr Table doenat rececive any new reocrds adn enqueued  jobs stopped

### DISCUSSION THREAD

**Discussion by engahmedtahaa** [jobrunr/jobrunr#642] [Q&A]
We’ve been using JobRunr scheduled jobs for a while with a SQL server 2019 database and we are using jobrunr 3.2.1

we figure out a very strange behavior the enqueued jobs don't work if the jobrunr_jobs table contains more than 1. 5 M records and when we delete some of the records it works. 

can anyone support us in this issue as we did not know what is the problem and there are no errors or exceptions appearing

**Comment by rdehuyss** [comment]
You are running on a very old version of JobRunr. 

First thing I would do is upgrade.
